package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "scratchcloud",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
Due to a feature change, a shift to 1.1.0 would probably be better. (third digit is a patch, second is a feature, and first is a refactor)